Projects/UOContent/Items/Skill Items/Misc/Bandage.cs

```diff
@@ -495,36 +495,33 @@ public static BandageContext BeginHeal(Mobile healer, Mobile patient)
                     seconds = 9.4 + 0.6 * ((double)(120 - dex) / 10);
                 }
             }
-            else
+            else if (Core.AOS && GetPrimarySkill(patient) == SkillName.Veterinary)
```

Comment:
No functional change in this file, just collapsed the else and if into an else-if.